watertap/costing/unit_models/mixer.py

```diff
@@ -133,7 +133,7 @@ def cost_naocl_mixer(blk, dosing_rate):
 
 
 def build_caoh2_cost_param_block(blk):
-    blk.cost = pyo.Param(
+    blk.CaOH2_cost = pyo.Param(
```

Comment:
Curious what inspired these changes to make `cost` more specific

Reply:
@MarcusHolly I wonder if we should just use `unit_cost` across the board. For chemicals,  $ / kg cost is technically a unit cost. Any cost per unit of quantity can be rendered a unit_cost. For pumps, unit_cost would be something like $/W, and so on.